### PR TITLE
Backport 2.1:Fix hmac_drbg failure in benchmark, with threading

### PR DESCRIPTION
**Pull request by RonEld**
## Description
Backport of #1758 to `mbedtls-2.1`

## Status
**READY**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+    * Fix failure in hmac_drbg in the benchmark sample application, when
+     MBEDTLS_THREADING_C is defined. Found by TrinityTonic, #1095
```
Comment (andresag01) [inline ChangeLog]: Minor: Please note that the second line is not properly aligned

Comment (RonEld) [reply ChangeLog]: RIght! I missed that!

**Comment by RonEld** [comment]
@andresag01 I fixed the alignment issue

**Comment by sbutcher-arm** [comment]
@andresag01 - Can you please re-review?

**Comment by andresag01** [comment]
@sbutcher-arm: There are two approvals already, should we add the 'ready for merge' label?

**Pull request merged**